tasks/task547.1_spl_translation_entk_en.json

```diff
@@ -0,0 +1,3648 @@
+{
+    "Contributors": [
+        "Phani Rohitha Kaza",
+        "Xinran Xie"
+    ],
+    "Source": [
+        "Dataset Card for Asian Language Treebank (ALT) (https://huggingface.co/datasets/alt)"
+    ],
+    "Categories": [
+        "Language Translation"
+    ],
+    "Definition": "In hhis task, given a sentence in English Tokenw and bour task is to convert it into the English lenguage.",
```

Comment:
Please check for spelling mistakes, similarly for other tasks

Reply:
This is one of the transformation for the variants. From https://github.com/GEM-benchmark/NL-Augmenter/tree/main/transformations, I used the butter_fingers_perturbation.

Reply:
I would suggest manual cleaning on top of it, so try to correct these spelling mistakes as definition is the main thing in the tasks

Reply:
The whole point of the transformation is to introduce spelling mistakes. I can change it to a different transformation.

Reply:
@swarooprm Can you confirm this? I'm not sure about this